### DISCUSSION TITLE
Autocompletion?

### DISCUSSION THREAD

**Discussion by xiaoweiz** [plotdevice/plotdevice#56] [Q&A]
Nice work, very slick. Is the code autocompletion under way?

**Comment by samizdatco** [comment]
There are two forms of autocompletion that are already working (to my knowledge). 

The simpler kind is akin to ‘snippets’ in TextMate: type a PlotDevice command's name (without parens) then hit tab and it will fill in the default arguments. You can also hit option-tab while midway through typing a word to get a more IntelliSense-style dropdown menu with completions matching the current word.

**Comment by x-raizor** [comment]
@samizdatco Wow! That a nice news. How we can add it to the manual?
